### PR TITLE
Add tests for gRPC tls and mTLS

### PR DESCRIPTION
**Pull request by bogdandrutu**

### CODE REVIEW

File: internal/helloworld/helloworld.proto [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// Copyright The OpenTelemetry Authors
```
Comment (tigrannajaryan) [inline internal/helloworld/helloworld.proto:1]: Why is this needed? Can we use one of OTLP request/responses for testing instead?

Comment (bogdandrutu) [reply internal/helloworld/helloworld.proto:1]: Good idea, I can do that.

Comment (bogdandrutu) [reply internal/helloworld/helloworld.proto]: Done.
